.github/workflows/ci.yml

```diff
@@ -33,6 +37,5 @@ jobs:
         env:
           STREAM_KEY: ${{ secrets.STREAM_KEY }}
           STREAM_SECRET: ${{ secrets.STREAM_SECRET }}
-        run: |
-          python setup.py install
-          make test
+          PYTHONPATH: ${{ github.workspace }}
```

Comment:
adding pip to path might not be needed anymore, good to update setup python

Reply:
let me see

Reply:
indeed. it works just fine without it. this `PYTHONPATH` is needed for pytest though. 🤷